includes/Modules/Analytics_4/Reset_Audiences.php

```diff
@@ -135,6 +135,7 @@ public function reset_audience_data() {
 						array(
 							'configuredAudiences' => null,
 							'didSetAudiences'     => false,
+							'isAudienceSegmentationWidgetHidden' => false,
```

Comment:
Not sure why this one hasn't been caught by phpcs, but `WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned` enforces all whitespaces in associated arrays always be aligned correctly, like we have in other arrays: https://github.com/google/site-kit-wp/blob/7295bd6732434687b0f9a66d1b8c9801bc4c0f6e/includes/Modules/Analytics_4/Settings.php#L90-L114
@jimmymadon, can you take a look why it hasn't been caught?

Reply:
As mentioned, manually aligning causes PHPCS to enforce the rule to keep things as they are.